_aggregations/metric/boxplot.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: default
+title: Boxplot
+parent: Metric aggregations
+grand_parent: Aggregations
+nav_order: 15
+---
+
+# Boxplot
+
+A boxplot aggregation calculates the statistical distribution of a numeric field. It provides summary of the data, including the following key statistics: minimum value, first quartile, median, third quartile, and maximum value.
+
+## Syntax
+
+The basic syntax for the boxplot aggregation is as follows:
+
+```json
+{
+  "aggs": {
+    "boxplot_agg_name": {
+      "boxplot": {
+        "field": "numeric_field"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+Replace `boxplot_agg_name` with a descriptive name for your aggregation and `numeric_field` with the name of the numeric field you want to analyze.
+
+## Example use case
```

Comment:
Developer: Please verify that the use case examples are relevant to the user and are accurate. If any changes are needed, please provide an updated example request. Thank you.